02LessonTwo/LessonTwo.js

```diff
@@ -1,14 +1,31 @@
 module.exports = function() {
 
-  this.whichSpecies = function (character) {
+this.whichSpecies = function (character) {
     // should return "dog" when character is 'scooby'
+    if (character === 'scooby') {
+    	return "dog";
+    }
     // should return "cat" when character is 'garfield'
+    else if (character === 'garfield') {
+    	return "cat";
+    }
     // should return "fish" when character is 'nemo'
+    else if (character === 'nemo') {
+    	return "fish";
+    }
     // should return false if character is anything else
+    else {	
+    	return false;
+
+    }
   }
 
   this.isEven = function (number) {
     // should return true is number is even (divisible by 2)
-  }
-
+ 	if (number % 2 === 0) {
```

Comment:
What kind of value does the expression number % 2 === 0 result in? What are its possible values, and in what circumstances do those possible values occur?